code/mission/missionmessage.cpp

```diff
@@ -403,6 +427,16 @@ void handle_legacy_backup_message(MissionMessage& msg, SCP_string wing_name) {
 	strcpy(msg.name, backup);
 }
 
+int lookup_mood(SCP_string const& name) {
+	for (auto i = Builtin_moods.begin(); i != Builtin_moods.end(); ++i) {
+		if (*i == name) {
```

Comment:
Should mood lookups be matched case-insensitively? If that is the case, then using `lcase_equal` as they are SCP_strings.